slow_odgi/mygfa.py

```diff
@@ -61,54 +60,56 @@ def __str__(self):
         )
 
 
+@dataclass(eq=True, frozen=True, order=True)
+class SegO:
+    """A specific orientation for a segment, referenced by name."""
+    name: str
+    orientation: bool
+
+    @classmethod
+    def parse(cls, s, o) -> "SegO":
+        return SegO(s, parse_orient(o))
+
+    def rev(self) -> "SegO":
+        return SegO(self.name, not self.orientation)
+
+    # We need two str methods because Links and Paths prefer SegOs to be
```

Comment:
But sadly, also some ugliness.
GFAs format paths like 
`P       x       1+,3+,5+ ...`
but links like
`L       1       +       3       + ...`
and now that the `1+` in the path and the `1       +` in the link are _the same thing_, we need to be able to format the same thing two different ways.

Reply:
Neato; seems reasonable enough to me!